crates/worker/src/partition/state_machine/mod.rs

```diff
@@ -2884,6 +2890,7 @@ impl<S> StateMachineApplyContext<'_, S> {
                         self.record_created_at,
                     );
 
+                info!("Starting invocation {invocation_id}, scheduler stats: {wait_stats:?}");
```

Comment:
Will be removed or downgraded to debug later, left temporarily to improve visibility until feature is complete.